docs/design/jet/025-jet-job-partition-pruning.md

```diff
@@ -123,9 +123,85 @@ try to choose only the required members.
 
 ### Scan partition pruning
 
-Partition pruning for IMap scan is a pretty simple optimization: we will extract the partition key 
-condition during SQL opt phase, pass it to a specialized processor supplier which will spawn 
-`ReadMapOrCacheP` with only required partitions to scan.
+Partition pruning for IMap scan supposed to be a pretty simple optimization: we should extract 
```

Comment:
"supposed to be a pretty simple optimization" - so isn't it simple?

Reply:
https://github.com/hazelcast/hazelcast/pull/25031/commits/4ed14f8d550e7c05d3c3510ca55a46881ea951cb